app/assets/stylesheets/styleguide/_font-stacks.scss

```diff
@@ -19,7 +19,7 @@
        local("Helvetica");
 }
 
-$NTA-Light:
+$nta-light:
```

Comment:
This is being renamed but I can't see it anywhere else in the PR - is it needed?

Reply:
Nope, I think I renamed it in a find and replace.